### PR TITLE
Lock GRPC low level particularity into a single wrapper

### PR DESCRIPTION
**Pull request by surabujin**

### CODE REVIEW

File: src-java/grpc-speaker/grpc-service/src/main/java/org/openkilda/grpc/speaker/controller/NoviflowController.java [hunk 1]
```diff
@@ -129,7 +129,7 @@
     public CompletableFuture<EnableLogMessagesResponse> enableLogOferrors(
             @PathVariable("switch_address") String switchAddress,
             @RequestBody LogOferrorsDto logOferrors) {
-        return grpcService.enableLogOferror(switchAddress, logOferrors);
+        return grpcService.enableLogOfError(switchAddress, logOferrors);
```
Comment (rozdy) [inline src-java/grpc-speaker/grpc-service/src/main/java/org/openkilda/grpc/speaker/controller/NoviflowController.java:132]: Rename method param as well.

File: src-java/grpc-speaker/grpc-service/src/main/java/org/openkilda/grpc/speaker/client/GrpcSession.java [hunk 2]
```diff
@@ -418,9 +395,36 @@ public void shutdown() {
...
+
+        GrpcResponseObserver<PacketInOutStats> observer = new GrpcResponseObserver<>(address, GET_PACKET_IN_OUT_STATS);
+        extendChain(
+                dummy -> stub.showStatsPacketInOut(ShowPacketInOutStats.newBuilder().build(), observer),
+                observer.future);
+        return observer.future
+                .thenApply(responses -> responses.stream().findFirst());
+    }
+
+    private synchronized void extendChain(Consumer<Void> action, CompletableFuture<?> operation) {
```
Comment (rozdy) [inline src-java/grpc-speaker/grpc-service/src/main/java/org/openkilda/grpc/speaker/client/GrpcSession.java:408]: Can we change `Consumer<Void>` to `Runnable` here?

**Review by rozdy — APPROVED** [review]

**Review by nikitamarchenko — APPROVED** [review]

File: src-java/grpc-speaker/grpc-service/src/main/java/org/openkilda/grpc/speaker/controller/NoviflowController.java (hunk 1 above)
Comment (surabujin) [reply src-java/grpc-speaker/grpc-service/src/main/java/org/openkilda/grpc/speaker/controller/NoviflowController.java]: done

File: src-java/grpc-speaker/grpc-service/src/main/java/org/openkilda/grpc/speaker/client/GrpcSession.java (hunk 2 above)
Comment (surabujin) [reply src-java/grpc-speaker/grpc-service/src/main/java/org/openkilda/grpc/speaker/client/GrpcSession.java]: yes, replaced

**Pull request merged**